modules/custom/cu_core/cu_core.info

```diff
@@ -26,6 +26,7 @@ dependencies[] = xmlsitemap_node
 dependencies[] = xmlsitemap_menu
 dependencies[] = cu_helpscout
 dependencies[] = cu_inactive_users
+dependencies[] = cu_users
```

Comment:
This can't be a real dependency because it causes secure_permission to get enabled which results in installs taking 10x as long.

https://github.com/CuBoulder/express/blob/dev/modules/custom/cu_users/cu_users.info#L6